RentACarWebAPI/Repositories/RentACarDbContext.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Microsoft.EntityFrameworkCore;
+using RentACarWebAPI.Models;
+
+namespace RentACarWebAPI.Repositories
+{
+    public class RentACarDbContext : DbContext
+    {
+        public RentACarDbContext(DbContextOptions options) : base(options) { }
+        
+        public DbSet<Car> Cars { get; set; }
+        public DbSet<Client> Clients { get; set; }
+        public DbSet<Rental> Rentals { get; set; }
+    }
```

Comment:
if you want you can add a unique constraint to the DNI column in Clients, check https://docs.microsoft.com/en-us/ef/core/modeling/indexes?tabs=data-annotations#index-uniqueness
I recommend using fluent API instead of data annotations so your models keep being database agnostic

Reply:
Great! Thanks for the advice! I will push it in a moment!